examples/stable_diffusion.py

```diff
@@ -621,20 +622,23 @@ def get_x_prev_and_pred_x0(x, e_t, index):
     x_prev = math.sqrt(a_prev) * pred_x0 + dir_xt #+ noise
     return x_prev, pred_x0
 
+  @TinyJit
+  def do_step(latent, timestep):
+    e_t = get_model_output(latent, timestep)
+    x_prev, _ = get_x_prev_and_pred_x0(latent, e_t, index)
```

Comment:
`index` here is from the loop on 638, and the calculations that use it use python math so it won't be jit captured and stays the same.